### ISSUE TITLE
Deploy Notebooks to Workspace failed, due to Missing argument "TARGET_PATH"

### ISSUE THREAD

**Issue by hangyao** [microsoft/azdo-databricks#13]
I tried to Deploy Notebooks to Workspace but failed due to Missing argument "TARGET_PATH".

Please find the screenshot below:

![image](https://user-images.githubusercontent.com/19369631/64816950-5cb51180-d56e-11e9-9133-561997a4619d.png)
![image](https://user-images.githubusercontent.com/19369631/64816989-6cccf100-d56e-11e9-8d1f-9885acac2671.png)
![image](https://user-images.githubusercontent.com/19369631/64817195-ee248380-d56e-11e9-9798-1b9bd0fa7db0.png)

I tried to use the default "/Shared" directory. Do I miss anything here?

Thank you!

**Comment by hangyao** [comment]
Just want to follow up. On line 14 it seems like the target path /Shared missing a quotation mark. Did it cause the issue?

**Comment by Isarien** [comment]
I have exactly the same issue.
Did you have a workaround ?

**Comment by hangyao** [comment]
@Isarien , I've switched to another tool in the marketplace, "Databricks Script Deployment Task by Data Thirst", and use its task "Databricks Notebooks deployment" to replace this one and it works for me.

**Comment by Isarien** [comment]
Yes I do the same. It works, but it's too bad that all the tasks do not work.

**Comment by Connell1** [comment]
It may depend on your build agent. I was getting the same error when running on a Windows agent, which is not listed in the supported agents in the docs. I switched to Ubuntu and it worked.

https://github.com/microsoft/azdo-databricks
